src/styles/blocks/card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $color-border-card;
+  border-radius: 5px;
+  margin-bottom: 50px;
+
+  &__image {
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    color: $color-blue-dark;
+    margin-bottom: 4px;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: $color-card-code-price;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    color: $color-card-code-price;
+  }
+
+  &__price-number {
+    font-weight: 700;
+    font-size: 16px;
+    color: $color-blue-dark;
+  }
+
+  &__link {
+    text-decoration: none;
+    background: $color-blue-light;
+    border-radius: 5px;
+    width: 166px;
+    height: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    color: #fff;
+    font-size: 14px;
+    font-weight: 700;
+    transition: 0.3s;
```

Comment:
specify the properties for which you use transition
by default it'll set transition for all properties, which can affect on website performance in bad way